### ISSUE TITLE
Chunk.createNode() crashes with an internal error

### ISSUE THREAD

**Issue by Sam54123** [rvandoosselaer/Blocks#49]
```
SEVERE: Uncaught exception thrown in Thread[jME3 Main,6,main]
java.lang.NoSuchMethodError: java.nio.FloatBuffer.flip()Ljava/nio/FloatBuffer;
        at com.rvandoosselaer.blocks.ChunkMesh.vector3fToBuffer(ChunkMesh.java:76)
        at com.rvandoosselaer.blocks.ChunkMesh.generateMesh(ChunkMesh.java:44)
        at com.rvandoosselaer.blocks.FacesMeshGenerator.createGeometry(FacesMeshGenerator.java:178)
        at com.rvandoosselaer.blocks.FacesMeshGenerator.lambda$createNode$1(FacesMeshGenerator.java:72)
        at java.util.HashMap.forEach(HashMap.java:1289)
        at com.rvandoosselaer.blocks.FacesMeshGenerator.createNode(FacesMeshGenerator.java:71)
        at com.rvandoosselaer.blocks.Chunk.createNode(Chunk.java:165)
        at org.scaffoldeditor.editor.editor3d.block.WorldManager.loadWorld(WorldManager.java:29)
        at org.scaffoldeditor.editor.editor3d.EditorApp.reload(EditorApp.java:91)
        at org.scaffoldeditor.editor.editor3d.EditorApp.simpleInitApp(EditorApp.java:67)
        at com.jme3.app.SimpleApplication.initialize(SimpleApplication.java:239)
        at com.jme3.system.lwjgl.LwjglAbstractDisplay.initInThread(LwjglAbstractDisplay.java:132)
        at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:213)
        at java.lang.Thread.run(Thread.java:748)
```
This is printed in the traceback when trying to generate the voxel mesh in some projects. This error seems seems to only occur in Java 1.8, which according to your build.gradle, you support. I'd be happy to provide more information to help you fix this.

**Comment by rvandoosselaer** [comment]
Are you building the sources yourself? This happens when you build the sources with JDK9+ and run it with JDK9<. Some new overriden methods are introduced in the ByteBuffer class in JDK9.

**Comment by rvandoosselaer** [comment]
Fix available in release [v1.5.1](https://github.com/rvandoosselaer/Blocks/releases/tag/v1.5.1)

**Comment by Sam54123** [comment]
I'm not building this my self. I'm using your Maven repo, and to my knowledge, JDK 8.

**Comment by rvandoosselaer** [comment]
The error should be solved in v1.5.1. Could you confirm?
